### DISCUSSION TITLE
The implementation with pytorch backend

### DISCUSSION THREAD

**Discussion by Xieyyyy** [lululxvi/deepxde#712] [General]
Hi, Lu Lu!
I don't find the demo with pytorch backend. In the examples, all demos are given by the tensorflow with default, how to switch it?
Or how can I generate a new demo with pytorch?
Thank you!

**Comment by lululxvi** [comment]
Look at the first line of the code
"""Backend supported: tensorflow.compat.v1, tensorflow, pytorch, jax, paddle"""
These are all supported backends for this example.

**Reply by Xieyyyy** [reply]
Sorry for bothering again, it seems that I don't well express my concerns.
I mean whether it can automatically recognize the backend based on the current environment? If I have several backends that simultaneously exist in the environment, how can I switch different backends?
Thank you.

**Reply by forxltk** [reply]
https://deepxde.readthedocs.io/en/latest/user/installation.html
